service/src/main/kotlin/fi/espoo/evaka/pis/service/PersonService.kt

```diff
@@ -357,6 +359,7 @@ data class PersonDTO(
     val lastName: String?,
     val email: String?,
     val phone: String?,
+    val backupPhone: String? = "",
```

Comment:
I don't think this field should be nullable since the data can't be null in backend.